ci/requirements-py3.9-min.yml

```diff
@@ -8,14 +8,14 @@ dependencies:
     - pytest-cov
     - pytest-mock
     - pytest-timeout
-    - python=3.8
+    - python=3.9
     - pytz
     - requests
     - pip:
         - dataclasses
```

Comment:
```suggestion
```

No longer needed for python >=3.7.  Looks like we forgot to remove that when dropping support for python 3.6.